### ISSUE TITLE
How to generate PCD maps in Autoware for NDT

### ISSUE THREAD

**Issue by ningfeiwang** [lgsvl/autoware-data#9]
Hello,

I wonder how to generate PCD maps for NDT localization in Autoware if I have a batch of PCD files and their real poses. I think the maps you provide is generated by driving the car in the simulator and collect the LiDAR frames data. I am not sure how I can stitch such pcds files together.

Thank you very much.

**Comment by martins-mozeiko** [comment]
I do not think you need to stitch files together. Autoware can load multiple pcd files for their map. Just select all of them you have.

**Comment by ningfeiwang** [comment]
Thank you very much. 

I am just curious whether Autoware will just simply load all points from different pcd fils and combine them together directly or make some special operation(like using filters) then put them together.

Also, what if all the pcd files have different original points? Can Autoware adds them together or I need to manually transform them into the same original points first?
